.github/workflows/nightly-pipeline.yaml

```diff
@@ -83,28 +83,28 @@ jobs:
     steps:
       - name: Checkout source code
         uses: actions/checkout@v2
-  
+
       - name: Install Rust toolchain
         uses: actions-rs/toolchain@v1
-      
+
       - name: Restore cache
         uses: ./.github/actions/restore-cache
         with:
           target-key: e2e
           cargo-key: e2e
           cache-version: v1
           cargo-targets: e2e-tests/target/
-      
+
       - name: Build binary and docker image
         run: |
           cd e2e-tests/
-          cargo build --release
```

Comment:
That was my initial idea but @bartoszjedrzejewski insited on having uniform builds across our pipelines so that if there is a bug due to the production profile we would catch it here. However, I would prefer faster e2e pipeline since I find it very improbable to bump into such a bug.

Reply:
I think integration tests (ie e2e tests) must be compiled using the same build settings as the production build, from the reason @mike1729 stated above.

Reply:
Ah but this build e2e client, not `aleph-node` test binary - in that case we can even (and I think now better) leave with non `--release`  , which is called `dev`: https://doc.rust-lang.org/cargo/reference/profiles.html#dev

Reply:
Changed to dev @fbielejec